### ISSUE TITLE
Add Neutrinet's new CalDav address

### ISSUE THREAD

**Issue by tbalthazar** [YoloSwagTeam/hackeragenda#123]
https://files.neutrinet.be/remote.php/dav/public-calendars/MQVZDHCFF8N0HQC6/

**Comment by hidrarga** [comment]
@Psycojoker Could you reopen the issue ? For some reasons, the calendar doesn't appear on the website :confused: Could you check that the changes are effeotive ? Thanks :)

**Comment by frju365** [comment]
No tested, but I reopen issue.

**Comment by hidrarga** [comment]
I just noticed that there was a Neutrinet event showing up in March. So the import is working...

The difference I see with other Neutrinet events is that it has a description. A quick look in the code shows that events without a description are skipped : https://github.com/YoloSwagTeam/hackeragenda/blob/master/agendas/be.py#L827 :facepalm: 

I have no idea why I did that at the time :confused: but the fix would be to just remove this condition and replace the line  https://github.com/YoloSwagTeam/hackeragenda/blob/master/agendas/be.py#L842 to handle event without a description.
